tests/data/console/controllers/fixtures/subdir/SecondFixture.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace yiiunit\data\console\controllers\fixtures\subdir;
+
+use yii\test\Fixture;
+use yiiunit\data\console\controllers\fixtures\FixtureStorage;
+
+class SecondFixture extends Fixture
+{
+    public function load()
+    {
+        FixtureStorage::$subdirSecondFixtureData[] = 'some data set for subdir/second fixture';
```

Comment:
Well, the string (the value) itself *can*, but *doesn't have to* be used in unit tests.
The property, howerer, [is used in unit-tests](https://github.com/yiisoft/yii2/pull/13553/files/719f6aee4856d2e5bbd9b44100955d9e34f94573#diff-410e9261d60564f099dc41e3dd30a538).
P.s. In fact, all the added fixtures and their usage in tests are *the same* as of existing fixtures.